### PR TITLE
`azurerm_sql_failover_group` document example added

### PR DESCRIPTION
**Pull request by yupwei68**
Fix https://github.com/terraform-providers/terraform-provider-azurerm/issues/8778

### CODE REVIEW

File: website/docs/r/sql_failover_group.html.markdown [hunk 1]
```diff
@@ -71,7 +71,7 @@ The following arguments are supported:
 
 * `databases` - A list of database ids to add to the failover group
 
--> **NOTE:** The failover group will create a secondary database for each database listed in `databases`. If the secondary databases need to be managed through Terraform, they should be defined as resources and a dependency added to the failover group to ensure the secondary databases are created first.
+-> **NOTE:** The failover group will create a secondary database for each database listed in `databases`. If the secondary databases need to be managed through Terraform, they should be defined as resources and a dependency added to the failover group to ensure the secondary databases are created first. Please refer to the detailed example which can be found in [the `./examples/mssql/failover_group` directory within the Github Repository](https://github.com/terraform-providers/terraform-provider-azurerm/tree/master/examples/mssql/failover_group)
```
Comment (tombuildsstuff) [inline website/docs/r/sql_failover_group.html.markdown:74]: this resource is `sql` but the example is `mssql`- does this want updating to be `sql` until the replacement resources are present?

File: examples/mssql/failover_group/variables.tf [hunk 2]
```diff
@@ -0,0 +1,11 @@
+variable "location" {
+  description = "The Azure location where the primary resources in this example should be created."
+}
+
+variable "location_alt" {
+  description = "The Azure location where the secondary resources in this example should be created."
+}
+
+variable "prefix" {
+  description = "The prefix used for all resources used by this NetApp Account"
```
Comment (tombuildsstuff) [inline examples/mssql/failover_group/variables.tf:10]: this isn't a NetApp Account

```suggestion
  description = "The prefix used for all resources created as a part of this example"

```

**Review by tombuildsstuff — CHANGES_REQUESTED** [review]
hi @yupwei68 

Thanks for this PR - I've taken a look through and left a couple of comments inline but if we can fix those up then this otherwise looks good to me

Thanks!

**Comment by yupwei68** [comment]
@tombuildsstuff Thanks for your comments. Please continue reviewing.

**Review by katbyte — APPROVED** [review]
Thanks @yupwei68 - LGTM 👍

**Pull request merged**